guiEditor/src/diagram/workbench.tsx

```diff
@@ -397,7 +434,28 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
             removeObservers();
         }, PointerEventTypes.POINTERUP);
 
-        scene.onPointerObservable.add(zoomFn, PointerEventTypes.POINTERWHEEL);
+
+        scene.onKeyboardObservable.add((k: KeyboardInfo, e: KeyboardEventTypes) => {

```

Comment:
should that be removed at a certain point? maybe the element unmount/deconstruction?